include/boost/url/authority_view.hpp

```diff
@@ -232,7 +232,7 @@ class BOOST_SYMBOL_VISIBLE
     {
         if(pos >= size())
             detail::throw_out_of_range(
-                BOOST_CURRENT_LOCATION);
+                BOOST_URL_CURRENT_LOCATION);
```

Comment:
if we get to pick the macro name why not `BOOST_URL_POS` for now

Reply:
I like `BOOST_URL_CURRENT_LOCATION` because then I don't have to remember two things. But both are fine.